persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAOImpl.kt

```diff
@@ -237,6 +239,12 @@ class ConversationDAOImpl(
             .map { it.map(conversationMapper::toModel) }
     }
 
+    override suspend fun getUnreadMessageCount(conversationID: QualifiedIDEntity): Long =
```

Comment:
why not make it return `Flow<Long>` so you'll know when it's changed?

Reply:
It is not needed for this feature yet